templates/js/www

```diff
@@ -58,5 +58,5 @@ function onError(error) {
  */
 
 function onListening() {
-  debug('Listening on port ' + server.address().port);
+  debug('Listening on port ' + (server.address().port || server.address()));
```

Comment:
Cool :) I was wondering how Node.js would have fit in the named pipe to `{host: ..., port: ...}`... just just not even giving an object, haha.